src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -396,11 +418,14 @@ export default Vue.extend<Data, Methods, Computed, Props>({
                 })
                 .then(({data}) => {
                     this.updateWorkflowReports(data.data);
-                    this.validationError = null;
+                    this.reportsValid = Array(data.data.length).fill(true);
+                    this.validationErrors = [];
                     this.isImportedReports = true
                 })
                 .catch((error) => {
-                    this.validationError = error.response.data?.errors;
+                    this.updateWorkflowReports([]);
+                    this.reportsValid = [];
```

Comment:
Instead of invalidating existing reportsValid, why not add another prop i.e csvReportValid to check for validation and you can then use reportValid || csvReportValid ?

Reply:
That's part of what will be happening when we implement separation of the two reports lists in https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2819 as per Alex's suggestion